predicators/datasets/__init__.py

```diff
@@ -45,6 +46,11 @@ def create_dataset(env: BaseEnv, train_tasks: List[Task],
     if CFG.offline_data_method == "demo+labeled_atoms":
         return create_ground_atom_data_from_labeled_txt(
             env, train_tasks, known_options)
+    if CFG.offline_data_method == "img_demos":  # pragma: no cover.
+        # NOTE: this below method is tested separately; it's just that testing
```

Comment:
the big reason is because the VLM is instantiated implicitly inside this method, but when testing, we don't want it to do that, but rather pass in a dummy VLM. Will update the comment to explain this.